ldif/ldif_test.go

```diff
@@ -181,4 +182,28 @@ func TestLDIFURL(t *testing.T) {
 	}
 }
 
-// vim: ts=4 sw=4 noexpandtab nolist
+var ldifMultiBlankLines = `# Organization Units
+dn: ou=users,dc=example,dc=com
+objectClass: organizationalUnit
+objectClass: top
+ou: users
+
+
+# searches for above empty line for dn but fails and errors out in this PR
+# Even though this is a valid LDIF file for ldapadd
```

Comment:
This looked a little familiar haha, these two comments aren't necessary anymore I don't think.